contributing.rst

```diff
@@ -306,30 +306,30 @@ The rest of the documentation assumes you are in the directory for the repo (the
   - The ``clone`` command creates a new directory inside the current one. So you do not need to create a new `odk-docs` directory first.
   - As noted above, we recommend a master :file:`odk` directory that holds your virtualenv directory and your git repo in two separate subdirectories. So you would be in that master :file:`odk` directory when you clone down the repo.
   - Double check that right folders are in the right places
-  
+
   .. code-block:: none
 
     - odk/
       - odkenv/
       - docs/
-	
+
 .. _upstream-the-docs:
 
 Set the Upstream Remote
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When you clone down a repo, the local copy calls your GitHub copy ``origin``. You should also set ``upstream`` as the name of the original, main GitHub repo.
+When you clone down a repo, the local copy calls your GitHub copy ``origin``. You should also set ``upstream`` as the name of the original master branch of the main GitHub repo.
```

Comment:
That change is not accurate.
 
'upstream` refers to the whole repo, not specifically the master branch

Reply:
Alright, thanks for the correction.
This one was actually my last edition to the file to make it more consistent with the alterations that I've made further in the code.